### DISCUSSION TITLE
Best way to add music to the app?

### DISCUSSION THREAD

**Discussion by TheFern2** [martpie/museeks#728] [Q&A]
I have playlists as folders on my computer. I would like to just drag and drop and have those playlists available. Is this possible?
From what I see, is that it wants files dragged into the library. Even if I make a playlist within the app and have the playlist empty and drag files, they go to the library instead of the playlist.

**Comment by martpie** [comment]
Sorry, for some reasons I forgot to reply. Creating the playlists on the fly when dropping them is planned but not done yet.

You're best importing your whole library from the settings, `.m3u` included, and Museeks will create the right playlists.
